files/sshd_config

```diff
@@ -10,24 +10,25 @@
 # possible, but leave them commented.  Uncommented options override the
 # default value.
 
+% if node.os != node.OS_FAMILY_DEBIAN or node.os_version > 10:
```

Comment:
this will create an empty sshd_config on non debian systems. i would not go this route, and instead have our sshd config in other systems as well (then at least ssh will work)

Reply:
Nope it will not.
It will remove the include if the OS is Debian and version lower than 11.

On Debian 10 there is an old SSH package that doesn't support Includes. ;)